Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml

```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: ClothingBackpackDuffelSyndicateBundle
+  id: ClothingBackpackDuffelSyndicateFilledRailgun
+  name: Decimator bundle
```

Comment:
```suggestion
  name: decimator bundle
```

Reply:
due to a renaming scheme this has been resolved